### DISCUSSION TITLE
Raw data from specific station in Home Assistant for Preschool in Japan

### DISCUSSION THREAD

**Discussion by DaleJP** [xspanger3770/GlobalQuake#243] [Q&A]
I just stumbled upon this application while trying to set up a warning system for my Nursery/Preschool in Saitama, Japan. It looks fantastic but is a little demanding to run 24/7.

When there is an earthquake, we often get early warnings on our phones, and I have a phone hooked into the PA system for this very purpose. The problem we have is that there are often false positives, and there is a moment when everyone freezes to see if the building is actually going to shake. I have set up an M5Stack Stick with an app called SeismoM5 that monitors accelerometer data and notifies me when there is shaking, but I wonder if we can use the data from our local station (JP JYT BHZ), which happens to be pretty close to us. I would love to display a number value and colour on a small display to indicate that actual shaking is being recorded. 

Would anyone be able to point me in the right direction to get the data from this specific station and display it in Home Assistant?

**Comment by nlucansk** [comment]
This is something I am also looking for ... I have no problem to self host GlobalQuake and it would be nice to have an API exposed that could be used to get such information out of it @xspanger3770 would it be possible?
